src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1271,6 +1271,19 @@ public static void DCJS_SerializationEvents()
         Assert.True(output.OnDeserializedMethodInvoked, "output.OnDeserializedMethodInvoked is false");
     }
 
+    [Fact]
+    public static void DCJS_DeserializeEmptyString()
+    {
+        var exception = Assert.Throws<SerializationException>(() =>
```

Comment:
Where does the exception actually come from... ReadObject?  If so, should we wrap just that call in the Assert?